addons/sale_project/__manifest__.py

```diff
@@ -1,14 +1,14 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+
 {
-    'name': "Sales - Project",
-    'summary': "Task Generation from Sales Orders",
+    'name': 'Project Auto-Generation on Sales Orders',
```

Comment:
I think this is what you mean?
```suggestion
    'name': 'Project Auto-Generation from Sales Orders',
```